### PR TITLE
STEM: Update Chapter 5 of 10203 Form Workflow to match the prototype #9876

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
https://github.com/department-of-veterans-affairs/va.gov-team/issues/9876

## Screenshots
Screen Shot 2020-06-25 at 11 27 53 AM
Screen Shot 2020-06-24 at 3 50 27 PM
Screen Shot 2020-06-24 at 3 50 22 PM
Screen Shot 2020-06-26 at 12 00 38 PM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: src/applications/edu-benefits/10203/content/directDeposit.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export const directDepositDescription = () => {
```
Comment (billfienberg) [inline src/applications/edu-benefits/10203/content/directDeposit.jsx:3]: According to the React docs, "Always start component names with a capital letter." - https://reactjs.org/docs/components-and-props.html

Comment (zurbergram) [reply src/applications/edu-benefits/10203/content/directDeposit.jsx:3]: This is code follows the standard within vets-website for form content. Please see https://github.com/department-of-veterans-affairs/vets-website/tree/master/src/applications/disability-benefits/all-claims/content

**Review by delli-gatti-michael — APPROVED** [review]

File: src/applications/edu-benefits/10203/content/directDeposit.jsx (hunk 1 above)
Comment (billfienberg) [reply src/applications/edu-benefits/10203/content/directDeposit.jsx:4]: That makes sense. If that's the existing pattern, then it's often better to stay consistent.

**Pull request merged**